layers/lldp.go

```diff
@@ -57,6 +62,21 @@ type LLDPChassisID struct {
 	ID      []byte
 }
 
+func (c *LLDPChassisID) TLVID() byte {
+	return byte(LLDPTLVChassisID)
+}
+func (c *LLDPChassisID) Serialize() []byte {
```

Comment:
do these functions need to be exported?

Reply:
I suppose we don't need to export these. I will keep in mind.